docs/content/topics/developers.md

```diff
@@ -108,297 +114,232 @@ seamlessly with Docker Desktop (Docker for Windows).
 To run lint checks:
 
 ```console
-$ make lint
-```
-
-To format the Go files:
-
-```console
-$ make format-go
+$ make lint-go
 ```
 
 To run the unit tests:
 
 ```console
-$ make test-unit
-```
-
-To re-run Go dependency resolution:
-
-```console
-$ make dep
+$ make test-unit-go
 ```
-
 ## Working with JS Code (for the Brigade Worker)
 
-To format the Javascript files:
+To lint the Javascript files:
 
 ```console
-$ make format-js
+$ make lint-js
 ```
 
 To run the tests:
 
 ```console
-$ make test-js
+$ make test-unit-js
 ```
 
-To re-run JS dependency resolution:
+To clear the JS dependency cache:
 
 ```console
-$ make yarn-install
+$ make clean-js
 ```
 
-(See `Running the Brigade-Worker Locally` below for live testing against a running instance.)
-
 ## Building Source
```

Comment:
I think this section probably needs to note that the builds for all Docker images occur using Kaniko in a container (because this is a convenient approach in CI where DinD is hard) and because these targets don't produce an image that's immediately available/usable locally, that the `hack-build-x` variants of those targets bypass the containerized Kaniko and build on the local system using plain Docker.